### ISSUE TITLE
[client] tooltips

### ISSUE THREAD

**Issue by CodeMyst** [pastemyst/pastemyst-v3#5]

**Comment by CodeMyst** [comment]
They should be done using TS to make sure the tooltips never go out of screen and cause overflow.
